In [1]:
# Compute the Pearson Correlation 
import pandas as pd
from scipy.stats import pearsonr

In [2]:
# Convert dataframe into series
list1 = [40,21,25,31,38,47]
list2 = [78,70,60,55,80,66]

In [3]:
# Apply the pearsonr()
corr, _ = pearsonr(list1, list2)
print('Pearson correlation: %.3f' % corr)

Pearson correlation: 0.347


Pearson correlation: 0.347 (Moderate Positive correlation)
Interpretaton:
As the Age increases, Weight increases 

In [4]:
# Convert dataframe into series
list1 = [20,10,23,5]
list2 = [30,5,29,10]
corr, _ = pearsonr(list1, list2)
print('Pearson correlation: %.3f' % corr)

Pearson correlation: 0.904
